### ISSUE TITLE
[Optimization

### ISSUE THREAD

**Issue by fwbrasil** [softwaremill/tapir#3577]
Tapir version: ***

Scala version: ***

**Describe the bug**

What is the problem?

**How to reproduce?**

Maybe you can provide code to reproduce the problem?

**Additional information**

**Comment by kciesielski** [comment]
Thanks, I wasn't aware of this possibility. It sounds like something that we might want to configure on the endpoint level instead of having a global `forkExecution` setting. This would require changes in a few places to properly propagate it, wdyt?

**Comment by fwbrasil** [comment]
Good point! I normally see this kind of configuration globally for a server but having endpoint-level control of the behavior could be very useful since typically only a few endpoints can benefit of the optimization in production.
